app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -60,6 +73,7 @@ def emails
       emails = [IdentityConfig.store.team_agnes_email]
       if report_date.day == 1
         emails << IdentityConfig.store.team_all_feds_email
+        emails << IdentityConfig.store.team_all_contractors_email
```

Comment:
I have kind of strong feelings about email, but: have we actually asked contractors (outside our team) if this is a thing they want? My GSA inbox is already a sea of stuff I never asked to receive, so I'm hesitant for us to add to that pile without a signal that people are actually going to look forward to receiving this.

Reply:
When I demoed the MKMR for the Engineering Huddle, there seemed to be general interest to receive the email.  Granted, that’s still some contractors speaking for all contractors, but we did have buy-in from those in attendance.

Reply:
That's good enough for me!